### ISSUE TITLE
Reproduced results on RSNA Pneumonia dataset

### ISSUE THREAD

**Issue by liangdima** [marshuang80/gloria#5]
Hi,
Thanks for your impressive work!

Recently, i tried to reproduce the classification results on the RSNA Pneumonia dataset. I used the released code and the pretrained model on your project and run the experiments follow your instruction. And surprisingly, i got the results even higher than the paper. Actually, i thought it might because of the randomness, but i tried to run several times with different random seed and all the results were superior, which makes me confused.

Do you think it's normal? Is there any improvement contained on your released pretrained model?

**Comment by marshuang80** [comment]
Hi there, thank you for taking the time to reproduce our results! I don't recall any changes or improvements in my released model, so it is surprising that you are getting better performance. Do you mind sharing the performance metric and your fine-tuning setup?
